src/XamlMath.Shared/Boxes/ZBox.cs

```diff
@@ -0,0 +1,38 @@
+using System.Linq;
+using XamlMath.Rendering;
+
+namespace XamlMath.Boxes
+{
+    public class ZBox : Box
```

Comment:
Maybe `LayeredBox`? What do you think? `ZBox` sounds kinda technical and wasn't obvious to me (I had to think for several seconds about that).

Reply:
Fixed in e9e1e1e.